bagging.md

```diff
@@ -1,22 +1,32 @@
 # Baggers
 
 - Getting set up as a Bagger
-  - Talk to your DataRescue guide to make sure you can become a Bagger.
+  - Apply to become a Bagger 
```

Comment:
"  - Apply to become a Bagger. At present, most events are using a Google form for this step. Please confirm with event organizers. " ?